packages/kcmsf/src/pages/entryBulk.tsx

```diff
@@ -0,0 +1,238 @@
+    });
+    return newErrors;
+  };
+
+  const sendData = () => {
+    const data = csvData.map((row) => {
+      const entry: Entry = {
+        teamName: row[0],
+        members: [row[1], row[2]],
+        isMultiWalk: row[3] === "歩行型" ? true : false,
+        category: row[4] as "Elementary" | "Open",
+      };
+      return entry;
+    });
+    console.log(data);
+    setisError(true);
+    //const json = JSON.stringify(data);
+    //なんかごにょごにょして後ろに渡す
+  };
+
+  const clear = () => {
+    setisError(false);
+    setCsvData([]);
+  };
+
+  return (
+    <>
+      <h1>一括エントリー</h1>
+      {csvData.length > 0 ? (
+        <>
+          <p>この内容で登録します</p>
+          <Box>{EntryTable(csvData, errors)}</Box>
+          <Button m={"2rem"} onClick={clear} variant="default">
+            リセット
+          </Button>
+          <Button m={"2rem"} onClick={sendData} disabled={iserror}>
+            <IconSend stroke={2} />
+            登録
+          </Button>
+        </>
+      ) : (
+        <Box
+          maw={620}
+          style={{
+            boxShadow: "4px 4px 6px rgba(0, 0, 0, 0.05)",
+            borderRadius: rem(15),
+          }}
+          mx="auto"
+          bd="2px solid gray"
+          p={rem(10)}
+        >
+          <Dropzone
+            onDrop={handleDrop}
+            onReject={() => (
+              <Text c={"red"} fw={700}>
+                CSVファイル以外のものは登録できません。
+              </Text>
+            )}
+            maxSize={5 * 1024 ** 2}
+            accept={["text/csv"]}
+          >
+            <Group
+              justify="center"
+              gap="xl"
+              mih={220}
+              style={{ pointerEvents: "none" }}
+            >
+              <Dropzone.Accept>
+                <IconUpload
+                  style={{
+                    width: rem(52),
+                    height: rem(52),
+                    color: "var(--mantine-color-blue-6)",
+                  }}
+                  stroke={1.5}
+                />
+              </Dropzone.Accept>
+              <Dropzone.Reject>
+                <IconX
+                  style={{
+                    width: rem(52),
+                    height: rem(52),
+                    color: "var(--mantine-color-red-6)",
+                  }}
+                  stroke={1.5}
+                />
+              </Dropzone.Reject>
+              <Dropzone.Idle>
+                <IconFileTypeCsv
+                  style={{
+                    width: rem(52),
+                    height: rem(52),
+                    color: "var(--mantine-color-dimmed)",
+                  }}
+                  stroke={1.5}
+                />
+              </Dropzone.Idle>
+              <div>
+                <Text size="xl" inline>
+                  ここにCSVファイルをドラッグ&ドロップしてください
+                </Text>
+              </div>
+            </Group>
+          </Dropzone>
+        </Box>
+      )}
+    </>
+  );
+};
+
+const EntryTable = (data: string[][], errors: boolean[][]) => {
```

Comment:
コンポーネントなので、
```suggestion
const EntryTable = (props: {data: string[][], errors: boolean[][]}) => {
```
のようにしてJSXとして使ってください。